src/base/bittorrent/session.cpp

```diff
@@ -4731,7 +4732,6 @@ void Session::handleAlert(const lt::alert *a)
         case lt::save_resume_data_failed_alert::alert_type:
         case lt::torrent_paused_alert::alert_type:
         case lt::torrent_resumed_alert::alert_type:
-        case lt::fastresume_rejected_alert::alert_type:
```

Comment:
Maybe keep this alert? It might report some other errors or I misunderstood something?